proposals/P0856.bs

```diff
@@ -0,0 +1,260 @@
+<pre class='metadata'>
+Title: Restrict Access Property for span and mdspan
+Abstract:  TODO Write abstract
+Shortname: P0856
+Revision: 0
+Date:   2018-02-12
+Editor: H. Carter Edwards, hcedwar@sandia.gov
+Editor: Christian Trott, crtrott@sandia.gov
+Editor: David S. Hollman, dshollm@sandia.gov
+Audience: LEWG
+URL: https://github.com/kokkos/array_ref/blob/master/proposals/P0856.bs
+Group: WG21
+Status: D
+Markup Shorthands: markdown yes
+Default Highlight: c++
+</pre>
+
+Revision History
+================
+
+## D0856r0
+
+2017-11-08 Albuquerque LEWG feedback for [[P0009r3]]:
+
+-   Generate paper for *restrict* access property for `span` and `mdspan`
+
+Motivation
+==========
+
+The restrict (non-aliasing) property is a widely useful mechanism to
+enable array access optimization. This functionality is part of the C
+standard, several C++ compilers' extensions, and native to FORTRAN
```

Comment:
I'll make notes here for later revisions.

"Fortran" hasn't been in all-caps since FORTRAN IV, if I remember right.